test/e2e/app-dir/actions/app/handler/route.js

```diff
@@ -1,5 +1,7 @@
 import { cookies } from 'next/headers'
 
+export const revalidate = 1
```

Comment:
Does this do anything since the handler uses `cookies()`?

Reply:
Shouldn't will remove